src/components/currentData.js

```diff
@@ -0,0 +1,77 @@
+import React, { PureComponent, PropTypes } from 'react'
+import { connect } from 'react-redux'
+import * as actions from '../actions/actionIndex'
+import reactCSS from 'reactcss'
+
+const styles = reactCSS({
+  'default': {
+    currentData: {
+      float: 'left',
+      width: '49.8%',
+      marginBottom: '0',
+      backgroundColor: '#ef4856',
+    },
+    temperatureText: {
+      fontSize: '50px',
+      marginBottom: '20px',
+      marginTop: '5px',
+    }
+  }
+})
+
+const { object, func} = PropTypes
+
+class CurrentData extends PureComponent {
+
+  static PropTypes = {
+    todaysSummary: object,
+    loadTodaysSummary: func,
```

Comment:
Either mark this prop as required, or add a default in `defaultProps`. To mark as required, change to: `loadTodaysSummary: func.isRequired,`